django/contrib/humanize/templatetags/humanize.py

```diff
@@ -32,7 +32,10 @@ def ordinal(value):
         return value
     if value < 0:
         return str(value)
-    if value % 100 in (11, 12, 13):
+    if value == 1:
+        # Translators: Ordinal format when value is 1
```

Comment:
This needs an ending dot:

```suggestion
        # Translators: Ordinal format when value is 1 (1st).
```